.github/workflows/cd-hl7v2-to-fhir.yml

```diff
@@ -8,5 +8,4 @@ jobs:
      uses: ./.github/workflows/cd-executor.yml
      secrets: inherit
      with:
-      working_dir: ./transformations/v2-to-fhirr4-service
-      service-name: hl7v2-to-fhir-transformation
\ No newline at end of file
+      working_dir: ./transformations/v2-to-fhirr4-service
```

Comment:
workflow has defined input as working-dir, but here working_dir is used.

Reply:
fixed with [6d9bf5b](https://github.com/wso2/open-healthcare-prebuilt-services/pull/45/commits/6d9bf5b2891bd6c99c43d27980ec381215719a9d)